docs/data/destinations/braze.md

```diff
@@ -9,17 +9,16 @@ Amplitude Data's Braze integration lets you stream your Amplitude event data str
 
 !!!note "Other Amplitude + Braze Integrations"
 
-    This integration streams Amplitude events to Braze. Amplitude offers other integrations with Braze: 
+    This integration streams Amplitude events to Braze. Amplitude offers other integrations with Braze:
```

Comment:
Should this description be updated now that we have identify forwarding? This is a question because our validation in the UI doesn't currently allow customers to only send users. Applies to other docs here too.

Reply:
hmm. maybe that's something we should fix in our UI though? There shouldn't be anything stopping us from sending users

Reply:
```suggestion
    This integration streams Amplitude events and users to Braze. Amplitude offers other integrations with Braze:
```
NIT: Either way, should we add a brief bit like this? I'm fine with whether you choose to do this or not, as long as it's consistent.